packages/wrangler/src/index.ts

```diff
@@ -600,10 +601,19 @@ export function createCLIParser(argv: string[]) {
 	);
 
 	// ai
-	wrangler.command("ai", "🤖 Manage AI models\n", (aiYargs) => {
+	wrangler.command("ai", "🤖 Manage AI models", (aiYargs) => {
 		return ai(aiYargs.command(subHelp));
 	});
 
+	// pipelines
+	wrangler.command(
+		"pipelines",
+		`🚰 Manage Worker Pipelines\n`,
```

Comment:
Is Pipelines starting as GA or open-beta? If the latter, this line needs the colourised "[open beta" decorator (see `wrangler vectorize`)